### PR TITLE
resolve #26

### LINKED ISSUE #26: update .gitignore

### PR DESCRIPTION
**Pull request by Guseyn**
Tune build process and make it work in travis

### CODE REVIEW

File: src/async/net/FreeRandomPort.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+'use strict'
+
+const { AsyncObject } = require('@cuties/cutie')
+const portfinder = require('portfinder')
+
+class FreeRandomPort extends AsyncObject {
```
Comment (g4s8) [inline src/async/net/FreeRandomPort.js]: @Guseyn I think it can be moved to `test/` directory

Comment (Guseyn) [reply src/async/net/FreeRandomPort.js]: @g4s8 we need FreeRandomPort in build process, not only in tests

**Comment by yegor256** [comment]
@rultor merge

**Comment by yegor256** [comment]
@rultor merge

**Pull request merged**